drizzlepac/runastrodriz.py

```diff
@@ -637,10 +646,10 @@ def process(inFile, force=False, newpath=None, num_cores=None, inmemory=True,
                 # Reset to state prior to applying a priori solutions
                 traceback.print_exc()
                 align_apriori = None
-                _trlmsg += "ERROR in applying a priori solution.\n"
+                super_logger.debug("ERROR in applying a priori solution.")
 
             if align_apriori is None or (not align_apriori[0]['alignment_verified']):
-                _trlmsg += "Resetting WCS to pipeline-default solutions..."
+                super_logger.debug("Resetting WCS to pipeline-default solutions...")
```

Comment:
Please keep Line 652 as INFO as it helps (at least me!) to know the a priori did not succeed, and the WCS will be an existing pipeline-default solution.